src/Network/Network/NetworkWatcher/NetworkConfigurationDiagnostic/NewAzureNetworkWatcherConfigurationDiagnosticProfileCommand.cs

```diff
@@ -0,0 +1,74 @@
+﻿// ----------------------------------------------------------------------------------
```

Comment:
@irrogozh please add the markdown help file for this cmdlet

Reply:
Added